include/inexor/vulkan-renderer/settings_decision_maker.hpp

```diff
@@ -21,35 +22,38 @@ class VulkanSettingsDecisionMaker {
 
 private:
     /// @brief Rates a graphcs card by its features.
-    /// @param graphics_card The graphics card.
-    /// @return A score which is greater or equal to 0.
+    /// @note The current implementation just sums up all available memory as a score.
+    /// @todo Implement additional graphics card rating criteria if desired.
+    /// @param graphics_card The graphics card which will be rated.
+    /// @return The graphics card's score which is greater or equal to 0.
     [[nodiscard]] std::size_t rate_graphics_card(const VkPhysicalDevice &graphics_card);
 
 public:
     /// @brief Automatically decides if a graphics card is suitable for this application's purposes.
     /// In order to be a suitable graphcs card for Inexor's purposes, it must fulfill the following criteria:
     /// - It must support a swapchain.
     /// - It must support presentation.
-    /// - Add more here if you want..
-    /// @note Add more checks to the validation mechanism if neccesary, e.h. check for geometry shader support.
+    /// @todo Add more checks to the validation mechanism if neccesary, e.h. check for geometry shader support.
     /// @param graphics_card The graphics card which will be checked for suitability.
-    /// @return True if the graphics card is suitable, false otherwise.
-    /// @warning Do not discriminate graphics cards which are not VK_PHYSICAL_DEVICE_TYPE_DISCRETE_GPU,
-    /// because this would deny some players to run Inexor on their machines!
+    /// @return 'true' if the graphics card is suitable, 'false' otherwise.
```

Comment:
```suggestion
    /// @return ``true`` if the graphics card is suitable, ``false`` otherwise.
```

Reply:
Done. I removed the `false otherwise` everywhere.